lib/Sema/TypeCheckAttr.cpp

```diff
@@ -3668,6 +3668,8 @@ static AbstractFunctionDecl *findAbstractFunctionDecl(
           missingAccessor = true;
       } else
         candidate = asd->getAccessor(AccessorKind::Get);
+    } else if (accessorKind != None) {
+        missingAccessor = true;
```

Comment:
minor, we can fix this later:
```suggestion
      missingAccessor = true;
```